src/cba-list-new/shadow.css

```diff
@@ -202,9 +201,133 @@ li .hasInfo::after
   margin-bottom: 1px;
   background: no-repeat var(--icon-project);
 }
+span.row
+{
+  height: 20px;
+}
+
+#group
+{
+  padding-left: 8px;
+  padding-right: 22px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  height: 21px;
+}
+
+.add-item
+{
+  width: 13px;
+  height: 12px;
+  display: inline-block;
+  background: no-repeat var(--icon-group-btn);
+}
+.add-item:after
+{
+  content: "";
+  width: 8px;
+  height: 8px;
+  background: no-repeat var(--icon-group-btn);
+}
+.add-item:hover
+{
+cursor: pointer;
+}
+.control
+{
+  padding-top: 2px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin-left: 22px;
+  height: 20px;
+}
+
+.subitem-btn
+{
+  margin-top: 3px;
```

Comment:
Hope you are not using margin for positioning the item in the middle, because this won't help if you change the font size.

Reply:
No, I've deleted this line, it comes from previous styles